### ISSUE TITLE
Select fields misaligned

### ISSUE THREAD

**Issue by idlethumbs** [Dogfalo/materialize#3559]
Select fields are aligned differently in chrome than in firefox(latest) and ie(11),
I have created a form with inline text that looks fine in chrome, the fields use default materialize padding and margins: 

ie: 
![image](https://cloud.githubusercontent.com/assets/9349874/17856601/3acd8ae2-6876-11e6-8cf6-d3d729be0e87.png)

ff: 
![image](https://cloud.githubusercontent.com/assets/9349874/17856615/475d4ed2-6876-11e6-83a3-7fa39814e6b2.png)

chrome (as intended):
![image](https://cloud.githubusercontent.com/assets/9349874/17856632/5b8f8726-6876-11e6-896c-51c4508bf252.png)

sample code: 

```
        <div class=" col s12">
            I'm looking to <div class='input-field'>
                <label class='active'> select</label>
                <select id="department" name="department" onchange="$('.form-part-2').slideDown();
                                        $('.search-container').addClass('started');">
                    <option selected disabled value="all">please select</option>
                    <option value="buy">buy</option>
                    <option value="rent">rent</option>
                    <!--<option value="3">find</option>-->
                </select>
            </div>
        </div>
```

**Comment by acburst** [comment]
Can you recreate this in a codependent or something? It's pretty hard to debug without CSS

**Comment by acburst** [comment]
Should be fixed 28b119a. Let me know if this works

**Comment by idlethumbs** [comment]
@acburst thanks this is a big improvement,
I have adjusted one additional adjustment to the margin as its a bit lower than other content 
```
.input-field.inline {
 margin-top: 0.4rem;
}
```
